src/libmongoc/src/mongoc/mongoc-client.c

```diff
@@ -1132,6 +1132,13 @@ _mongoc_client_new_from_topology (mongoc_topology_t *topology)
    BSON_ASSERT (topology);
    BSON_ASSERT (topology->valid);
 
+#ifndef MONGOC_ENABLE_SSL
+   if (mongoc_uri_get_tls (topology->uri)) {
+      MONGOC_ERROR ("Can't create SSL client, SSL not enabled in this build.");
+      return NULL;
+   }
+#endif
```

Comment:
This added block looks unintended.

Reply:
I think that's an artifact of me removing it and then replacing it during a rebase-- it does appear in master.

Reply:
This block is _not_ present in the [master branch](https://github.com/mongodb/mongo-c-driver/blob/f6e6a7025d3c65a24e3c32c2763dadf3ef5b804a/src/libmongoc/src/mongoc/mongoc-client.c#L1134), please remove.